Unix/configure

```diff
@@ -2471,9 +2470,14 @@ __ROOT=$root
 export OUTPUTDIR=$outputdir
 export ROOT=$root
 export ENABLE_ULINUX=$enable_ulinux
+export ENABLE_NATIVE_KITS=$enable_native_kits
 export DISABLE_SSL_0_9_8=$disable_ssl_0_9_8
 
 OMI_CONFIGURE_QUALS=--enable-microsoft --disable-makefile-gen
+ifeq (\$(ENABLE_NATIVE_KITS),1)
+  OMI_CONFIGURE_QUALS=--enable-microsoft --disable-makefile-gen --enable-native-kits
```

Comment:
So this is a generated Makefile (generated by configure). What this says is: If, on the initial configure line, you did a `--enable-native-kits`, pass that on down if you need to reconfigure for SSL 0.9.8 and SSL 1.0.0.